src/styles/theme/oc-visibility.scss

```diff
@@ -9,28 +9,28 @@
 }
 
 /* Phone landscape and bigger */
-@media (min-width: $breakpoint-small) {
+@media (min-width: $oc-breakpoint-small-default) {
   .oc-hidden\@s {
     display: none !important;
   }
 }
 
 /* Tablet landscape and bigger */
-@media (min-width: $breakpoint-medium) {
+@media (min-width: $oc-breakpoint-medium-default) {
   .oc-hidden\@m {
     display: none !important;
   }
 }
 
 /* Desktop and bigger */
-@media (min-width: $breakpoint-large) {
+@media (min-width: $oc-breakpoint-large-default) {
   .oc-hidden\@l {
     display: none !important;
   }
 }
 
 /* Large screen and bigger */
-@media (min-width: $breakpoint-xlarge) {
+@media (min-width: $oc-breakpoint-xlarge) {
```

Comment:
Why no `-default` in this case?

Reply:
only variables with variations have a default flag.